sdk/keyvault/azkeys/client.go

```diff
@@ -56,7 +55,7 @@ func NewClient(vaultUrl string, credential azcore.TokenCredential, options *Clie
 
 	genOptions.PerRetryPolicies = append(
 		genOptions.PerRetryPolicies,
-		shared.NewKeyVaultChallengePolicy(credential, runtime.NewPipeline("azkeys", "0.1.0", nil, nil, genOptions)),
```

Comment:
I see the pending 0.1.1 release of the internal module requires this pipeline parameter.  What's the plan for releasing that?

Reply:
The internal module will be released before `azsecrets`/`azkeys`, then the change will be made in both keys and secrets